### DISCUSSION TITLE
最新3.12.0版添加阿里云盘open失败

### DISCUSSION THREAD

**Discussion by KimmyLueng** [AlistGo/alist#3580] [Q&A]
Failed init storage: Post "https://api.nn.ci/alist/ali_open/token": x509: certificate signed by unknown authority

**Comment by 14960** [comment]
同

**Reply by drw2005** [reply]
我这是获取不到令牌了，提示：TooManyRequests

是不是因为申请的太火爆啊

**Reply by nxjonson** [reply]
去掉s，用http://api.nn.ci/alist/ali_open/token

**Reply by 14960** [reply]
TooManyRequests这个错误，你多刷新几次当前页面就可以了

**Reply by loylou** [reply]
今天突然遇到的这个问题。求解

**Comment by mustundead** [comment]
有解决方法吗
